File: examples/lh-server/go/04-conditionals/tasks.go
package main

import "fmt"

func FetchUser(userID string) string {
	if userID == "obiwan" || userID == "padme" || userID == "satine" {
		return "COMLINK"
	}
	return "HOLOGRAM"
}

func SendComLink(userID, message string) string {
	return reportMessage("comlink", userID, message)
}

func SendHologram(userID, message string) string {
	return reportMessage("hologram", userID, message)
}

func reportMessage(method, userID, message string) string {
	result := fmt.Sprintf("sent %s %s to user %s", method, message, userID)
	fmt.Println(result)
	return result
}

File: examples/lh-server/python/04-conditionals/tasks.py
async def fetch_user(user_id: str) -> str:
    return "COMLINK" if user_id in ["obiwan", "padme", "satine"] else "HOLOGRAM"


async def send_comlink(user_id: str, message: str) -> str:
    result = "sent comlink " + message + " to user " + user_id
    print(result)
    return result


async def send_hologram(user_id: str, message: str) -> str:
    result = "sent hologram " + message + " to user " + user_id
    print(result)
    return result
